examples/cvode/CXX_onemkl/cvRoberts_blockdiag_onemkl.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char *argv[])
   if (argc > 3) output = (atoi(argv[3])) ? true : false;
 
   // Create an in-order GPU queue
-#if SYCL_LANGUAGE_VERSION >= 2020
+#if SYCL_LANGUAGE_VERSION >= 2020 && !defined(SUNDIALS_SYCL_2020_UNSUPPORTED)
```

Comment:
Where does `SUNDIALS_SYCL_2020_UNSUPPORTED` come from?

Reply:
OK, but I dont see where it is defined in CMake. Is this just passed in as a compiler flag? Should it be documented?